.scalafix.conf

```diff
@@ -1,3 +1,11 @@
 rules = [
-  NoInfer
+  Disable
```

Comment:
Can managed files or protobuf sources be excluded here?

Reply:
managed sources are ignored by scalafix by default, the problem is that all non-primitive fields are `Option[T]` and we do `.get` on them in several places, esp. in tests.